data/releases.yaml

```diff
@@ -653,31 +653,31 @@ Releases:
-    url: https://github.com/nats-io/nats-streaming-server/releases/download/v0.7.2/nats-streaming-server-v0.7.2-darwin-amd64.zip
-  - name: nats-streaming-server-v0.7.2-linux-386.zip
-    url: https://github.com/nats-io/nats-streaming-server/releases/download/v0.7.2/nats-streaming-server-v0.7.2-linux-386.zip
-  - name: nats-streaming-server-v0.7.2-linux-amd64.zip
-    url: https://github.com/nats-io/nats-streaming-server/releases/download/v0.7.2/nats-streaming-server-v0.7.2-linux-amd64.zip
-  - name: nats-streaming-server-v0.7.2-linux-arm6.zip
-    url: https://github.com/nats-io/nats-streaming-server/releases/download/v0.7.2/nats-streaming-server-v0.7.2-linux-arm6.zip
-  - name: nats-streaming-server-v0.7.2-linux-arm7.zip
-    url: https://github.com/nats-io/nats-streaming-server/releases/download/v0.7.2/nats-streaming-server-v0.7.2-linux-arm7.zip
-  - name: nats-streaming-server-v0.7.2-linux-arm64.zip
-    url: https://github.com/nats-io/nats-streaming-server/releases/download/v0.7.2/nats-streaming-server-v0.7.2-linux-arm64.zip
-  - name: nats-streaming-server-v0.7.2-windows-386.zip
-    url: https://github.com/nats-io/nats-streaming-server/releases/download/v0.7.2/nats-streaming-server-v0.7.2-windows-386.zip
-  - name: nats-streaming-server-v0.7.2-windows-amd64.zip
-    url: https://github.com/nats-io/nats-streaming-server/releases/download/v0.7.2/nats-streaming-server-v0.7.2-windows-amd64.zip
+  - name: nats-streaming-server-v0.9.2-darwin-amd64.zip
+    url: https://github.com/nats-io/nats-streaming-server/releases/download/v0.9.2/nats-streaming-server-v0.9.2-darwin-amd64.zip
+  - name: nats-streaming-server-v0.9.2-linux-386.zip
+    url: https://github.com/nats-io/nats-streaming-server/releases/download/v0.9.2/nats-streaming-server-v0.9.2-linux-386.zip
+  - name: nats-streaming-server-v0.9.2-linux-amd64.zip
+    url: https://github.com/nats-io/nats-streaming-server/releases/download/v0.9.2/nats-streaming-server-v0.9.2-linux-amd64.zip
+  - name: nats-streaming-server-v0.9.2-linux-arm6.zip
+    url: https://github.com/nats-io/nats-streaming-server/releases/download/v0.9.2/nats-streaming-server-v0.9.2-linux-arm6.zip
+  - name: nats-streaming-server-v0.9.2-linux-arm7.zip
+    url: https://github.com/nats-io/nats-streaming-server/releases/download/v0.9.2/nats-streaming-server-v0.9.2-linux-arm7.zip
+  - name: nats-streaming-server-v0.9.2-linux-arm64.zip
+    url: https://github.com/nats-io/nats-streaming-server/releases/download/v0.9.2/nats-streaming-server-v0.9.2-linux-arm64.zip
+  - name: nats-streaming-server-v0.9.2-windows-386.zip
+    url: https://github.com/nats-io/nats-streaming-server/releases/download/v0.9.2/nats-streaming-server-v0.9.2-windows-386.zip
+  - name: nats-streaming-server-v0.9.2-windows-amd64.zip
+    url: https://github.com/nats-io/nats-streaming-server/releases/download/v0.9.2/nats-streaming-server-v0.9.2-windows-amd64.zip
   - name: nats-streaming-server-v0.8.0-beta-darwin-amd64.zip
```

Comment:
Remove from here down to line before Docker Image link please.

Reply:
Ok - so remove all beta 0.8.0 releases. Do you want the beta release information removed from the text box at the top left of the page too?